spec/models/service_template_filter_spec.rb

```diff
@@ -48,4 +48,46 @@ def build_model
       @request.miq_request_tasks.count.should eql(1)
     end
   end
+
+  context "#automate_result_include_service_template?" do
+    before do
+      TestClass = Class.new do
+        include ServiceTemplate::Filter
+        def initialize(workspace)
+          @workspace = workspace
+        end
+      end
+    end
+
+    after do
+      Object.send(:remove_const, :TestClass)
+    end
+
+    let(:workspace) { instance_double("MiqAeEngine::MiqAeWorkspace", :root => options) }
+    let(:test_class) { TestClass.new(workspace) }
```

Comment:
More specifically, this let, which is the only user of TestClass, never gets called.